README.md

```diff
@@ -0,0 +1,72 @@
+# elasticache
+
+Provides an ElastiCache Replication Group resource.
+
+Example Usage
+----------------
+
+```hcl
+module "elasticache" {
+  source = "git@github.com:techservicesillinois/terraform-aws-elasticache"
+
+  cluster_name       = "service-name"
+
+  network_configuration {
+    vpc  = "name-of-vpc"
+    tier = "public"
+    security_group_names = "service-redis-clients"
+  }
+}
+```
+
+Argument Reference
+-----------------
+
+The following arguments are supported:
+
+* `cluster_name` - (Required) The name or identifier of the Replication group.  
+
+* `security_group_ids` - (Required) One or more Amazon VPC security groups associated with this replication group. Use this parameter only when you are creating a replication group in an Amazon Virtual Private Cloud
```

Comment:
The variable `security_group_ids` is commented out in `variables.tf`, so this bullet should be removed.